README.md

```diff
@@ -64,20 +65,47 @@ Signature algorithms:
 * SHA512withECDSAinP1363Format
 * RSASSA-PSS
 
-KeyPairGenerator algorithms:
+KeyPairGenerator:
 * EC
 * RSA
 
+KeyGenerator:
+* AES
+
 KeyAgreement:
 * ECDH
 
-SecureRandom algorithms:
-* ACCP's SecureRandom uses AWS-LC's DRBG implementation, which is described [here](https://github.com/awslabs/aws-lc/blob/main/third_party/jitterentropy/README.md) and [here](https://github.com/awslabs/aws-lc/blob/725625435158150ef21e0a4dab6fa3aca1ef2d2c/crypto/fipsmodule/rand/rand.c#L36-L60).
+SecretKeyFactory:
+* HkdfWithHmacSHA1
+* HkdfWithHmacSHA256
+* HkdfWithHmacSHA384
+* HkdfWithHmacSHA512
+
+SecureRandom:
+* ACCP's SecureRandom uses AWS-LC's DRBG implementation, which is described [here](https://github.com/awslabs/aws-lc/blob/main/third_party/jitterentropy/README.md) and [here](https://github.com/awslabs/aws-lc/blob/725625435158150ef21e0a4dab6fa3aca1ef2d2c/crypto/fipsmodule/rand/rand.c#L36-L60). Please refer to [system properties](https://github.com/corretto/amazon-corretto-crypto-provider#other-system-properties) for more information.
 
-KeyFactory algorithms:
+KeyFactory:
 * EC
 * RSA
-* ACCP's SecureRandom uses AWS-LC's DRBG implementation.
+
+AlgorithmParameters:
+* EC. Please refer to [system properties](https://github.com/corretto/amazon-corretto-crypto-provider#other-system-properties) for more information.
+
+
+# Notes on ACCP-FIPS
+ACCP-FIPS is a variation of ACCP which uses AWS-LC built in FIPS-mode.
+AWS-LC is undergoing FIPS validation testing by an accredited lab and,
+upon completion, will be submitted to NIST for certification.
+This process can take a long time, so we provide ACCP-FIPS for
```

Comment:
I would rephrase this sentence to highlight that we are releasing ACCP-FIPS to allow users to do early performance testing before we are granted our certificate.  I would also add a note in here to call out where future updates for the status of ACCP-FIPS will be.  Blog post?